cvmfs/cvmfs_suid_helper.cc

```diff
@@ -94,7 +94,7 @@ static void Mount(const string &path) {
       }
       systemd_unit = cvmfs_suid::EscapeSystemdUnit(resolved_path);
     }
-    ExecAsRoot("/bin/systemctl", "start", systemd_unit.c_str(), NULL);
+    ExecAsRoot("/bin/systemctl", "restart", systemd_unit.c_str(), NULL);
```

Comment:
I vaguely remember that "restart" does not do anything if the unit is not already started. But I might be wrong about that or perhaps systemd changed behavior.  Integration tests should tell.

Reply:
From the `systemctl` manpage:
> restart PATTERN...
>    Stop and then start one or more units specified on the command line. If the units are not running yet, they will be started.
>
> try-restart PATTERN...
>     Stop and then start one or more units specified on the command line if the units are running. This does nothing if units are not running.

so using `restart` should be OK, and it seems to work for me (on Fedora 34).